src-test/tests.hs

```diff
@@ -84,4 +87,17 @@ tests =
       assertReject "nstars" "9********" nstars
       assertReject "nstars" "10*********" nstars
       assertReject "nstars" "11**********" nstars
+
+  , testCase "isAlive" $
+      let alive p  = isAlive (coerce p)
+          dead p   = not (alive p)
+          feed p s = (foldl consume p s) in do
+        assertBool "" $ dead empty
+        assertBool "" $ dead (empty <|> empty)
+        assertBool "" $ dead (some empty)
+        assertBool "" $ alive (many empty)
```

Comment:
Can someone check, whether this makes sense? Usually, I would expect `dead` here. What is the meaning of `empty.*`?